settings.gradle

```diff
@@ -1 +1 @@
-include 'core', 'plugin'
\ No newline at end of file
+include 'core', 'plugin', 'examples:app-example'
```

Comment:
this was intentionally not a subproject (it's also used to verify that maven uploads work, and also as a standard example template, so it should always work if you copy-paste it outside the repo)

But perhaps that's the wrong way of doing it?